.travis.yml

```diff
@@ -1,4 +1,16 @@
 language: go
 
 go:
-  - 1.12.14
+  - 1.12.x
+  - 1.13.x
+  - 1.14.x
```

Comment:
should we support all these version? normally we specify one version

Reply:
Because Vungo is an open source project, we should not assume caller's go version. But our private applications like jaeger always use specified go version.